.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: go
+sudo: required
+
+services:
+  - docker
+go:
+  - 1.9
+  - "1.10"
```

Comment:
nit: why double quotes for this.

Reply:
without quotes travis CI interprets go version "1.10" as "1.1"